src/main/java/org/jabref/logic/bst/VM.java

```diff
@@ -1174,25 +1174,17 @@ private void strings(Tree child) {
 
     public static class BstEntry {
 
-        private final BibEntry entry;
+        public final BibEntry entry;
```

Comment:
Then this triggers a follow-up question (@LinusDietz): is it ok to make variables with mutable types (like `Map<>`) public via `public final` ? Because then outside code can modify the content of a class to their liking (which I guess is against some of the principles of object-oriented programming).

Reply:
This is effective Java #39. "Make defensive copies when needed".

> You must program defensively, with the assumption that clients of your class will do their best to destroy its invariants.

However, did not see that being done much, so I am curios to the opinion of @LinusDietz